crates/kitsune_p2p/kitsune_p2p/tests/common/harness.rs

```diff
@@ -120,6 +121,7 @@ impl KitsuneTestHarness {
         self.legacy_host_api.drain_events().await
     }
 
+    #[allow(dead_code)]
```

Comment:
Sorry yes, this is commented in the tests at the moment because they need more work. Thanks for cleaning up the warnings